storage/innobase/include/ut0ut.h

```diff
@@ -305,6 +305,33 @@ operator<<(
 	return(lhs);
 }
 
+/** This is a wrapper class, used to print any number in IEC style */
+struct bytes_iec {
+ explicit bytes_iec(unsigned long long t): m_val(t) {}
+ const unsigned long long m_val;
+};
+
+/** Like hex operator above, except for bytes_iec */
+inline
+std::ostream&
+operator<<(
+	std::ostream&	lhs,
+	const bytes_iec&	rhs)
+{
+  static const char *sizes[]= {"B", "KiB", "MiB", "GiB", "TiB", "PiB",
+                              "EiB", "ZiB", "YiB"};
+  size_t i = 0;
+  double d = (double) rhs.m_val;
```

Comment:
Monty wants to omit space before any assignment operators.
Please do not use C-style casts when an alternative exists.
```c++
  size_t i= 0;
  double d= static_cast<double>(rhs.m_val);
```
Could `iec_bytes` simply define a member function that would return `double`?